### PR TITLE
Clarify alpha arg description in CosineDecay docstring

### PR DESCRIPTION
**Pull request by kysolvik**
Related to #21772 and #21827 

Descriptions of `alpha` in CosineDecay docstring conflict: in explanation text it says it applies to `warmup_target` or `initial_learning_rate` (if `warmup_target` is None), but in the parameter description it says `alpha` is a fraction of `initial_learning_rate`. 

Actual behavior is that `alpha` is applied to the initial learning rate *after* warmup. If there is no warmup, that is `initial_learning_rate`. Otherwise, it is `warmup_target`. 

Updated parameter description to clarify this.

#21827 fixed the explanation text but not the parameter description.

### CODE REVIEW

File: keras/src/optimizers/schedules/learning_rate_schedule.py [hunk 1]
```diff
@@ -651,7 +651,8 @@ def decayed_learning_rate(step):
         initial_learning_rate: A Python float. The initial learning rate.
         decay_steps: A Python int. Number of steps to decay over.
         alpha: A Python float. Minimum learning rate value for decay as a
-            fraction of `initial_learning_rate`.
+            fraction of `warmup_target` or, if `warmup_target` is None,
+            `initial_learning_rate`.
```
[the remark this answers is not in the document]
Comment (kysolvik) [reply keras/src/optimizers/schedules/learning_rate_schedule.py:655]: I'll leave it up to whoever reviews/merges this PR to decide whether to apply this suggestion. It would match CosineDecayRestarts, but for CosineDecay with warmup (unlike CosineDecayRestarts) there are two possible minimum learning rates: the initial_learning_rate and the final learning rate after decay, so I think this change could cause confusion. Maybe for maximum clarity: "Minimum learning rate value after decay as a fraction..."?

**Review by hertschuh — APPROVED** [review]
Thanks for fixing this!
